spec/system/whats_new_page_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe 'Whats new page' do
+  include_context 'with user'
+
+  context 'when exisiting user' do
```

Comment:
Great work, only comments are about the clarity of the spec contexts. I think `when exisiting user` (also typo) could be improved but this shouldn't prevent it merging.

Reply:
typo has been fixed, but it's still not the best name